### DISCUSSION TITLE
How run conatiners as root with local imported images?

### DISCUSSION THREAD

**Discussion by igoryan17** [containers/podman#10537] [Q&A]
I perform importing ```podman image import ...``` and can run container with image name ```podman run {some_image_name}```. But with **sudo** i can run only from search registers from  _/etc/containers/registries.conf_. It's possible run containers as root with local images?

**Comment by rhatdan** [comment]
Please show us the exact commands you are executing.  Running sudo podman import, sudo podman run, should definitely work.

**Reply by igoryan17** [reply]
`podman import my-image.tar my-image`
Then there is `docker.io/library/my-image` record in column 'Repository' after running `podman images`
`sudo podman run -it my-image bash`
and after that i can choose only registers, but without sudo works fine

**Reply by igoryan17** [reply]
I have performed podman save and then run with sudo from archive. It's works fine.
